test/ex_doc/formatter/html_test.exs

```diff
@@ -102,4 +123,28 @@ defmodule ExDoc.Formatter.HTMLTest do
     assert HTML.pretty_codeblocks(iex_detected_without_class) ==
           "<pre><code class=\"iex elixir\">iex&gt; max(4, 5)"
   end
+
+  test "run generates index.html and normalized options" do
+    # 1. Check for default [main: "overview"]
+    # 2. Check for output dir having trailing "/" stripped
+    for dir <- ["#{output_dir}", "#{output_dir}/", "#{output_dir}//"] do
```

Comment:
I think we can test just only one of those. The trailing stuff is already checked on the unit test below. Generally speaking, we should avoid integration tests (that generates the docs and goes through everything) for every possible combination. The unit tests are enough. :)